_data/fr.yml

```diff
@@ -243,16 +243,16 @@ quality:
     introduction: 'Demandez à 100 développeurs ce qui fait une bonne bibliothèque et vous obtiendrez bon nombre de réponses différentes. Comme cela est long et fastidieux, <em>nous</em> avons érigé une liste de règles simples à suivre qui vous permettront de coder une bibliothèque de qualité. Celle-ci doit :'
     button: 'Voir l’architecture d’un projet'
     rules:
-        - 'Utiliser <code>League</code> comme espace de nom pour la norme <a href="http://www.php-fig.org/psr/psr-4/">PSR-4</a>. Le code source est entièrement contenu dans le dossier <code>src</code>;'
-        - 'Adhérer à la norme <a href="http://www.php-fig.org/psr/psr-2/fr" target="_blank">PSR-2</a> comme style d’écriture de code ;'
-        - '&Ecirc;tre reprise sur Packagist avec <code>league</code> comme espace de nom vendeur ;'
-        - 'Contenir des tests unitaires. Le but étant d’avoir une bibliothèque testée à plus de 80% pour la version 1.0 ;'
-        - 'Documenter tous les élements du code source avec des <a href="http://www.phpdoc.org/docs/latest/references/phpdoc/index.html" target="_blank">DocBlock</a> ;'
-        - 'Utiliser <a href="http://semver.org/lang/fr/" target="_blank">la gestion sémantique de version</a> pour gérer les mises à jour ;'
-        - 'Maintenir un fichier <a href="https://keepchangelog.com" target="_blank">CHANGELOG</a>'
-        - 'Utiliser <a href="https://travis-ci.org/" target="_blank">Travis-CI</a> pour continuellement vérifier les normes d’écriture et lancer ses tests unitaires ;'
+        - 'Utiliser <code>League</code> comme espace de nom pour la norme <a href="http://www.php-fig.org/psr/psr-4/">PSR-4</a>. Le code source est entièrement contenu dans le dossier <code>src</code>.'
+        - 'Adhérer à la norme <a href="http://www.php-fig.org/psr/psr-2/fr" target="_blank">PSR-2</a> comme style d’écriture de code.'
+        - '&Ecirc;tre reprise sur Packagist avec <code>league</code> comme espace de nom vendeur.'
```

Comment:
Haha, I had the same problem! :)

I really need to just block Chrome translating from GitHub entirely.